docker/scripts/install_network_utils.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+apt-get -y update
```

Comment:
Can we add a quick comment explaining what this is - is that it's a script that is intended to be manually run with sudo as part of debugging a container?